CHANGELOG.md

```diff
@@ -30,6 +30,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Document modified
 - `print_normalized_matrix` method modified
 - `normalized_table_calc` function modified
+- `setup.py` modified
```

Comment:
Could you please update `CHANGELOG.md` with :
+ Python 3.8 added to travis and appveyor configs
+ Dockerfile modified

Reply:
Thanks for your comment ;-)
Done!